### DISCUSSION TITLE
UserId not found.

### DISCUSSION THREAD

**Discussion by TiranBaghdasaryan** [egramtel/tdsharp#144] [Q&A] [answered]
How to send message other people when I trying get user by userId I get - user not found, how to get users from telegram database, have any way ?

**Comment by ForNeVeR** [comment, marked as answer]
Please share the code you're using to get the user.

**Reply by TiranBaghdasaryan** [reply]
Thank you for response, but I already found the solution.
So for sending message other users we should firstly get CHAT by username _client.SearchPublicChatAsync()
Secondly we should use user id from CHAT to send message.
